test/e2e/common_test.go

```diff
@@ -367,6 +369,18 @@ func (p *PodmanTestIntegration) RunLsContainer(name string) (*PodmanSessionInteg
 	return session, session.ExitCode(), session.OutputToString()
 }
 
+// RunNginxWithHealthCheck runs the alpine nginx container with an optional name and adds a healthcheck into it
+func (p *PodmanTestIntegration) RunNginxWithHealthCheck(name string) (*PodmanSessionIntegration, string) {
+	var podmanArgs = []string{"run"}
+	if name != "" {
+		podmanArgs = append(podmanArgs, "--name", name)
+	}
+	podmanArgs = append(podmanArgs, "-dt", "-P", "--healthcheck-command", "CMD-SHELL curl http://localhost/ || exit 1", nginx)
```

Comment:
The `|| exit 1` is unnecessary: curl already exits nonzero on ECONNREFUSED.

(Unless something specifically requires an exit code of 1, in which case I think that merits an explanatory comment)

Reply:
fixed